src/convenience.jl

```diff
@@ -320,6 +321,19 @@ function GetCryptoContext(object::Union{Ciphertext})
     GetCryptoContext(GetCryptoContextProxy(object[]))
 end
 
+"""
+    GetEncodingParameters(object::Union{Ciphertext})
+
+Return the [`EncodingParams`](@ref) used by the given `object` that is is a subtype of `CryptoObject`.
+
+Currently, this is only implemented for [`Ciphertext`](@ref).
+
+See also: [`EncodingParams`](@ref), [`Ciphertext`](@ref)
+"""
+function GetEncodingParameters(object::Union{Ciphertext})
+    GetEncodingParameters(object[])
+end
+
```

Comment:
Since it is not doing anything but unwrapping the smart pointer, `GetEncodingParameters` should be added to the list of auto-unwrapped methods in `src/convenience.jl`, and the implementation be deleted here.

Only the documentation part
```suggestion
"""
    GetEncodingParameters(object::Union{Ciphertext})

Return the [`EncodingParams`](@ref) used by the given `object` that is is a subtype of `CryptoObject`.

Currently, this is only implemented for [`Ciphertext`](@ref).

See also: [`EncodingParams`](@ref), [`Ciphertext`](@ref)
"""
GetEncodingParameters

```
should be moved to `src/documentation.jl`.

Reply:
I added auto-unwrapping, but it works only explicit with `Ciphertext`, with `CryptoObject` I cannot use `GetEncodingParameters` on `Ciphertext`, see https://github.com/sloede/OpenFHE.jl/actions/runs/7884613806/job/21513974038?pr=38#step:7:73

Reply:
OK, then fix it as you see fit :-)